data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/v1/beacon/PostAttesterSlashing.java

```diff
@@ -0,0 +1,90 @@
+import static tech.pegasys.teku.infrastructure.http.HttpStatusCodes.SC_BAD_REQUEST;
+import static tech.pegasys.teku.infrastructure.http.HttpStatusCodes.SC_OK;
+
+import com.fasterxml.jackson.databind.JsonMappingException;
+import io.javalin.http.Context;
+import io.javalin.plugin.openapi.annotations.HttpMethod;
+import io.javalin.plugin.openapi.annotations.OpenApi;
+import io.javalin.plugin.openapi.annotations.OpenApiContent;
+import io.javalin.plugin.openapi.annotations.OpenApiRequestBody;
+import io.javalin.plugin.openapi.annotations.OpenApiResponse;
+import tech.pegasys.teku.api.DataProvider;
+import tech.pegasys.teku.api.NodeDataProvider;
+import tech.pegasys.teku.api.schema.AttesterSlashing;
+import tech.pegasys.teku.beaconrestapi.handlers.AbstractHandler;
+import tech.pegasys.teku.beaconrestapi.schema.BadRequest;
+import tech.pegasys.teku.provider.JsonProvider;
+import tech.pegasys.teku.statetransition.validation.InternalValidationResult;
+
+public class PostAttesterSlashing extends AbstractHandler {
+  public static final String ROUTE = "/eth/v1/beacon/pool/attester_slashings";
+  private final NodeDataProvider nodeDataProvider;
+
+  public PostAttesterSlashing(final DataProvider dataProvider, final JsonProvider jsonProvider) {
+    super(jsonProvider);
+    this.nodeDataProvider = dataProvider.getNodeDataProvider();
+  }
+
+  public PostAttesterSlashing(final NodeDataProvider provider, final JsonProvider jsonProvider) {
+    super(jsonProvider);
+    this.nodeDataProvider = provider;
+  }
+
+  @OpenApi(
+      path = ROUTE,
+      method = HttpMethod.POST,
+      summary = "Submit attester slashing object",
+      tags = {TAG_V1_BEACON},
+      description =
+          "Submits attester slashing object to node's pool and if passes validation node MUST broadcast it to network.",
+      requestBody = @OpenApiRequestBody(content = {@OpenApiContent(from = AttesterSlashing.class)}),
+      responses = {
+        @OpenApiResponse(
+            status = RES_OK,
+            description =
+                "Attester Slashing has been successfully validated, added to the pool, and broadcast."),
+        @OpenApiResponse(
+            status = RES_BAD_REQUEST,
+            description =
+                "Invalid attester slashing, it will never pass validation so it's rejected"),
+        @OpenApiResponse(status = RES_INTERNAL_ERROR),
+      })
+  @Override
+  public void handle(final Context ctx) throws Exception {
+    try {
+      final AttesterSlashing attesterSlashing =
+          jsonProvider.jsonToObject(ctx.body(), AttesterSlashing.class);
+      InternalValidationResult result =
+          nodeDataProvider.postAttesterSlashing(attesterSlashing).join();
+      if (result.equals(InternalValidationResult.IGNORE)
+          || result.equals(InternalValidationResult.REJECT)) {
+        ctx.status(SC_BAD_REQUEST);
+        ctx.result("Invalid attester slashing, it will never pass validation so it's rejected");
```

Comment:
can't do this and get decent json.
```ctx.result(BadRequest.badRequest(jsonProvider, "Invalid attester slashing, it will never pass validation so it's rejected."));```

Reply:
Done.